src/main/java/io/pravega/connectors/flink/AbstractStreamingWriterBuilder.java

```diff
@@ -24,8 +24,8 @@
 public abstract class AbstractStreamingWriterBuilder<T, B extends AbstractStreamingWriterBuilder> extends AbstractWriterBuilder<B> {
 
     // the numbers below are picked somewhat arbitrarily at this point
```

Comment:
Is the above comment in the code still true?   Are these values consistent with the underlying Pravega client's defaults?  The new values are radically different, which suggests a problem, i.e. that we don't really understand them.

Reply:
> // the numbers below are picked somewhat arbitrarily at this point

I forgot to remove this comment (will update). 

Yes, the values are consistent with Pravega's default and  I have posted my observation here https://github.com/pravega/flink-connectors/issues/126